### DISCUSSION TITLE
forget password link

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5588] [Help]
hi everyone
how are you ?
as you see in the below image it is suppose when i click on forget password and enter the email it is suppose that i get an email but i get nothing how can i solve this problem 

![WhatsApp Image 2021-03-21 at 4 18 54 PM](https://user-images.githubusercontent.com/77009503/111913934-b4440f80-8a78-11eb-9d17-050c4d95e64a.jpeg)

**Comment by edwardch** [comment]
Did you try to debug this to see why?

You probably need to configure your `SmtpSettings` in your app config.

**Reply by samaremad** [reply]
i edit the settings in web.config as following :

`<mailSettings>
      <smtp deliveryMethod="SpecifiedPickupDirectory"  from="workasp90@gmail.com " ></smtp>
    </mailSettings>`

can you please help me with that

**Comment by wezmag** [comment]
You will probably find the email in your project App_Data/Mail/ folder.

if you want to send it, you will need to change the settings in web.config as below:

``` xml
<smtp deliveryMethod="Network" from="workasp90@gmail.com">
        <network host="yoursmtpserver.com" />
</smtp>
```

**Reply by wezmag** [reply]
BTW, for development, I used a dummy stmp server.
https://github.com/rnwood/smtp4dev
There is an old v2 Windows GUI version.
You can change the network host to `localhost`, if you use the dummy server.
